src/SearchRoom/SearchRoom.css

```diff
@@ -0,0 +1,42 @@
+  h1 {
+    margin:0;
+  }
```

Comment:
This also appears in ViewRoom.css, would it be alright to move this to Main.css?

Reply:
Yeah I’ll move it